code/modules/research/designs/AI_module_designs.dm

```diff
@@ -24,8 +24,10 @@
 	departmental_flags = DEPARTMENT_BITFLAG_SCIENCE
 
 /datum/design/board/onehuman_module
-	name = "OneHuman Module"
-	desc = "Allows for the construction of a OneHuman AI Module."
+	//bubber edit begin - makes OneHuman Suitable for Safeguard
+	name = "OneCrew Module"
+	desc = "Allows for the construction of a OneCrew AI Module."
+	//bubber edit end
```

Comment:
Modularize this please

Reply:
Once this is modular Ill be happy to merge